Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/moth.yml

```diff
@@ -0,0 +1,1210 @@
+# Moth Foods, the three C's: cheese, coleslaw, and cotton
+# Base ingredients
+- type: entity
+  name: herby cheese
+  parent: FoodBakingBase
+  id: FoodMothHerbyCheese
+  description: As a staple of mothic cuisine, cheese is often augmented with various flavours to keep variety in their diet.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - cheesy
+      - leafy
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: herbycheese
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 7
+        - ReagentId: Protein
+          Quantity: 2
+        - ReagentId: Cream
+          Quantity: 1
+#Tastes like cheese, herbs
+
+- type: entity
+  name: mothic salad
+  parent: FoodBowlBase
+  id: FoodMothSaladBase
+  description: A basic salad of cabbage, red onion and tomato.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - cabbage
+      - onion
+      - tomato
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: mothsaladbase
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 6
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 2
+        - ReagentId: Vitamin
+          Quantity: 4
+
+# Pesto
+- type: entity
+  name: pesto
+  parent: FoodBakingBase
+  id: FoodMothPesto
+  description: A combination of salt, herbs, garlic, oil, and pine nuts. Frequently used as a sauce for pasta or pizza, or eaten on bread.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - garlic
+      - nutty
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: pesto
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: Vitamin
+          Quantity: 3
+
+# Bechamel
+- type: entity
+  name: bechamel sauce
+  parent: FoodBakingBase
+  id: FoodMothBechamel
+  description: A classic white sauce common to several European cultures.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - creamy
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: bechamel
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: Vitamin
+          Quantity: 3
```

Comment:
Bechamel is among the most decadent and fatty things out there, it's an instant heart attack eaten alone. Also, 25u liquids + flour into 3u of reagents is not great. I would probably make it contain Cream and Oil and a touch of Nutriment, 20u total. And given it's pretty much always an intermediate ingredient (since it's a sauce), I'd rather see it as a liquid, but that's something for later.

Reply:
I could solve that by creating a condiment/reagent for the Bechamel/Pasto and have the microwave recipe create a bowl filled with 20u of said reagent.
Then I can make the recipes use only 10u of those sauce so it makes it less costly overall.

Reply:
Awesome.